turbo.json

```diff
@@ -33,6 +33,7 @@
     "NEXT_PUBLIC_POSTHOG_KEY",
     "SUPABASE_URL",
     "SUPABASE_SERVICE_ROLE",
-    "NEXT_PUBLIC_POSTHOG_HOST"
+    "NEXT_PUBLIC_POSTHOG_HOST",
+    "RESEND_KEY"
```

Comment:
Should we add this to the .env.example too?

How can we add the env so AWS EC2 also have it?

Reply:
> How can we add the env so AWS EC2 also have it?
In this moment we access EC2 and config envs manually